e2e/provision/README.md

```diff
@@ -194,63 +223,84 @@ In any case, you now can push the package with the labels applied back to the
 repository:
 
 ```bash
-kpt alpha rpkg push -n default mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868 regional
+kpt alpha rpkg push -n default mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868 /tmp/nephio-e2e-lab/regional
 ```
 
+<details>
+<summary>The output is similar to:</summary>
+
+```console
+[RUNNING] "gcr.io/kpt-fn/apply-replacements:v0.1.1" 
+[PASS] "gcr.io/kpt-fn/apply-replacements:v0.1.1"
+```
+</details>
+
 Finally, you propose and approve the package.
 
 ```bash
 kpt alpha rpkg propose -n default mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868
 ```
 
-The output is similar to:
-```
+<details>
+<summary>The output is similar to:</summary>
+
+```console
 mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868 proposed
 ```
+</details>
 
 ```bash
 kpt alpha rpkg approve -n default mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868
 ```
 
-The output is similar to:
-```
+<details>
+<summary>The output is similar to:</summary>
+
+```console
 mgmt-08c26219f9879acdefed3469f8c3cf89d5db3868 approved
 ```
+</details>
 
 ConfigSync running in the management cluster will now pull out this new
 package, creating all the resources necessary to provision a Kind cluster and
 register it with Nephio. This will take about five minutes or so.
 
-## Step 6: Check Regional cluster installation
+### Step 2: Check Regional cluster installation
 
 You can check if the cluster has been added to the management cluster:
 
 ```bash
 kubectl get clusters
 ```
 
-The output is similar to:
-```
+<details>
+<summary>The output is similar to:</summary>
+
+```console
 NAME       PHASE         AGE     VERSION
 regional   Provisioned   52m     v1.26.3
 ```
+</details>
 
 To access the API server of that cluster as well, you
 need to get the `kubeconfig` file for it. To retrieve the file, you
 pull it from the Kubernetes Secret, and decode the Base64 encoding:
 
 ```bash
-kubectl get secret regional-kubeconfig -o jsonpath='{.data.value}' | base64 -d > regional-kubeconfig
+kubectl get secret regional-kubeconfig -o jsonpath='{.data.value}' | base64 -d > /tmp/nephio-e2e-lab/regional-kubeconfig
+export KUBECONFIG=$HOME/.kube/config:/tmp/nephio-e2e-lab/regional-kubeconfig
 ```
 
 You can then use it to access the workload cluster directly:
 
 ```bash
-kubectl --kubeconfig regional-kubeconfig get ns
+kubectl get ns --context regional-admin@regional
```

Comment:
How about retaining the `kubectl --kubeconfig regional-kubeconfig get ns` as well for folks used to using the `--kubedconfig` option?

Reply:
Both methods work, but the [official documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) makes use of contexts, the major advantage of context is that they can be persistent.

Reply:
Ok for context

Reply:
I am going to merge this as is, but at some point it would be nice to name the contexts so we can just say `--context regional` and `--context edge01` etc.